columnq/Cargo.toml

```diff
@@ -68,31 +69,11 @@ rusqlite = "0"
 
 [features]
 default = ["rustls"]
-rustls = [
-    "hyper-rustls",
-    "reqwest/rustls-tls",
-    "deltalake/s3",
-    "deltalake/azure",
-    "yup-oauth2/hyper-rustls",
-]
-native-tls-vendored = [
-    "reqwest/native-tls-vendored",
-    "hyper-tls/vendored",
-    "deltalake/s3",
-    "deltalake/azure",
-    "yup-oauth2/hyper-tls",
-]
-native-tls = [
-    "reqwest/native-tls",
-    "hyper-tls",
-    "deltalake/s3",
-    "deltalake/azure",
-    "yup-oauth2/hyper-tls",
-]
+rustls = ["hyper-rustls", "reqwest/rustls-tls", "deltalake/s3", "deltalake/azure", "yup-oauth2/hyper-rustls"]
```

Comment:
nit, could we change it back to multi-line formatting to keep diffing easier to follow?

Reply:
sure